### PR TITLE
new template action: TLS protocol version

### PR DESCRIPTION
**Pull request by henrydcase**
### 1. What does this change do, exactly?
Introduces new template action to get TLS protocol version

### 2. Please link to the relevant issues.
None

### 3. Which documentation changes (if any) need to be made because of this PR?
Template action should be documented here: https://caddyserver.com/docs/template-actions

### 4. Checklist

- [x]  I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I am willing to help maintain this change if there are issues with it later

### CODE REVIEW

**Comment by henrydcase** [comment]
let me know if this is interesting. then I'll add tests if needed

File: caddyhttp/httpserver/tplcontext.go [hunk 1]
```diff
@@ -448,6 +449,20 @@ func (c Context) AddLink(link string) string {
 	return ""
 }
 
+// Resturns TLS protocol versions
```
Comment (francislavoie) [inline caddyhttp/httpserver/tplcontext.go]: Typo:
```suggestion
// Returns TLS protocol versions
```

File: caddyhttp/httpserver/tplcontext.go [hunk 2]
```diff
@@ -448,6 +449,20 @@ func (c Context) AddLink(link string) string {
 	return ""
 }
 
+// Returns TLS protocol version
+func (c Context) TLSVersion() string {
+	switch c.Req.TLS.Version {
```
Comment (mholt) [inline caddyhttp/httpserver/tplcontext.go]: `c.Req.TLS` might be nil -- resulting in a panic.

Comment (henrydcase) [reply caddyhttp/httpserver/tplcontext.go]: Good point. I'll add tests and fix this. Thnx!

**Comment by henrydcase** [comment]
I've decided to change implementation and reuse "GetSupportedProtocolName" to get a string with protocol version. Pro - Context.TLSVersion doesn't need to be changed after TLS 1.3 is added, Cons - instead of getting string like "1.2" we get "tls1.2"

**Comment by mholt** [comment]
Thank you!

**Pull request merged**